vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -61,31 +61,28 @@ type internal FSharpCheckerProvider
             // When the F# background builder refreshes the background semantic build context for a file,
             // we request Roslyn to reanalyze that individual file.
             checker.BeforeBackgroundFileCheck.Add(fun (fileName, extraProjectInfo) ->  
-               async {
-                try 
-                    match extraProjectInfo with 
-                    | Some (:? Workspace as workspace) -> 
-                        let solution = workspace.CurrentSolution
-                        let documentIds = solution.GetDocumentIdsWithFilePath(fileName)
-                        if not documentIds.IsEmpty then 
-                            let docuentIdsFiltered = documentIds |> Seq.filter workspace.IsDocumentOpen |> Seq.toArray
-                            for documentId in docuentIdsFiltered do
-                                Trace.TraceInformation("{0:n3} Requesting Roslyn reanalysis of {1}", DateTime.Now.TimeOfDay.TotalSeconds, documentId)
-                            if docuentIdsFiltered.Length > 0 then 
-                                analyzerService.Reanalyze(workspace,documentIds=docuentIdsFiltered)
-                    | _ -> ()
-                with ex -> 
-                    Assert.Exception(ex)
+                async {
+                    try 
+                        match extraProjectInfo with 
+                        | Some (:? Workspace as workspace) -> 
+                            let solution = workspace.CurrentSolution
+                            let documentIds = solution.GetDocumentIdsWithFilePath(fileName)
+                            if not documentIds.IsEmpty then 
+                                let documentIdsFiltered = documentIds |> Seq.filter workspace.IsDocumentOpen |> Seq.toArray
+                                for documentId in documentIdsFiltered do
+                                    Trace.TraceInformation("{0:n3} Requesting Roslyn reanalysis of {1}", DateTime.Now.TimeOfDay.TotalSeconds, documentId)
```

Comment:
Why not just a regular timestamp?

Reply:
@brettfo this line came from:   
https://github.com/Microsoft/visualfsharp/pull/3777/files/93ebd231f81eb6de79fa42a48cebd268e4831e54#diff-cdf0a4fbb7615b8029d78133924259c3L73
just indented deeper.